frontends/web/src/locales/en/app.json

```diff
@@ -27,13 +27,15 @@
       "alreadyExists": "The account already exists.",
       "invalidAddress": "Please enter a valid Ethereum address",
       "xprivEntered": "WARNING: You entered an extended private key. Please enter an extended public key and keep your private keys safe.",
-      "xpubInvalid": "Extended Public Key malformatted.",
-      "xpubWrongNet": "Extended Public Key not valid for the selected account."
+      "xpubInvalid": "Extended Public Key malformatted."
     },
     "extendedPublicKey": "Extended Public Key",
     "scriptType": "Script Type",
     "submit": "Add Account",
-    "title": "Add Account"
+    "title": "Add Account",
+    "warning": {
+      "xpubWrongNet": "The extended public key does not match the selected account but, since not all apps export extended public keys correctly, we created the account anyway."
```

Comment:
Better wording:

`"Warning: the Extended Public Key format indicates that it might not valid for the selected account type. It you have trouble with the account, try the other account types."`